packages/dashboard/src/components/Wizard/examples/WizardExample.Base.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import { ISubwayNavNodeProps, SubwayNavNodeState } from '@uifabric/dashboard';
+import { IWizardStepProps, IWizardStepAction } from '@uifabric/dashboard/lib/components/Wizard/Wizard.types';
+import { getNextStep } from './SetupWizard.Util';
+import { setSubwayState } from '../../SubwayNav/examples/SubwayNav.Util';
+import { Label, PrimaryButton, DefaultButton } from 'office-ui-fabric-react';
+
+export interface IWizardExampleBaseState {
+  steps: IWizardStepProps[];
+  currentStepId: string;
+}
+export class WizardExampleBase<T extends IWizardExampleBaseState> extends React.Component<{}, T> {
+  protected steps: IWizardStepProps[] = [];
+
+  protected getTestSteps = (): IWizardStepProps[] => {
+    const testHeader = <Label>Wizard Title</Label>;
+    const testFooter = (
+      <>
+        <DefaultButton>Back</DefaultButton>
+        <PrimaryButton>Next</PrimaryButton>
+        <DefaultButton>Exit Wizard</DefaultButton>
```

Comment:
Do you want to link the on click with actions in the test example, at least the mainAction?

Reply:
Sure thing